react/react2/github-users-search/src/components/GithubResults.js

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+import { GithubContext } from './GithubContext';
+
+const GithubResults = () => {
+  const { loading, error, users } = useContext(GithubContext);
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return <div>Error fetching ({error})</div>;
+  }
+
+  if (users.length === 0) {
+    return <div>No results...</div>;
+  }
+
+  return (
+    <ul>
+      {users.map((user) => (
```

Comment:
You can try destructuring the user object.

Reply:
Done:) thank you so much  for reviewing my code :)